src/components/generic-elements/alert-component/alert-component.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+alert-component {
+  @import './node_modules/bootstrap/scss/bootstrap.scss';
```

Comment:
Have we proved that this works once we have packaged our library?  I'm not very good scss, but I'm curious as to whether this path stays intact once webpack gets ahold of it.